test/Base.t.sol

```diff
@@ -428,14 +428,22 @@ abstract contract Base_Test is Assertions, Events, StdCheats, V2CoreUtils {
             (lockupLinear, defaults.createWithRange(), defaults.permit2Params(defaults.PER_STREAM_AMOUNT()))
         );
         bytes memory response = aliceProxy.execute(address(target), data);
-        return abi.decode(response, (uint256));
+        return decodeAsUint256(response);
     }
 
     function createWithRange(LockupLinear.CreateWithRange memory params) internal returns (uint256) {
         bytes memory data = abi.encodeCall(
             target.createWithRange, (lockupLinear, params, defaults.permit2Params(defaults.PER_STREAM_AMOUNT()))
         );
         bytes memory response = aliceProxy.execute(address(target), data);
-        return abi.decode(response, (uint256));
+        return decodeAsUint256(response);
+    }
+
+    function decodeAsUint256(bytes memory input) private pure returns (uint256) {
+        return abi.decode(input, (uint256));
+    }
+
+    function decodeAsUint256Array(bytes memory input) private pure returns (uint256[] memory) {
+        return abi.decode(input, (uint256[]));
```

Comment:
Adding these functions increases the number of lines of code - let's apply the principle of KISS and remove them